templates/node--funding--full.html.twig

```diff
@@ -0,0 +1,211 @@
+                        <span class="funding-ct--summary-section--label wdn-sans-serif">Documents:</span>
+                        <ul class="funding-ct--summary-section--list">
+                            {% for i, value in resources_documents['#items'] %}
+                                <li>{{ resources_documents[i] }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    {% if node.field_funding_resources_links %}
+                        <span class="funding-ct--summary-section--label wdn-sans-serif">Links:</span>
+                        <ul class="funding-ct--summary-section--list">
+                            {% for i, value in resources_external_links['#items'] %}
+                                <li>{{ resources_external_links[i] }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                </section>
+            {% endif %}
+        </div>
+
+        <div class="bp768-wdn-col-five-eighths">
+            {# taxonomy section that is visible on desktop #}
+            <section class="funding-ct--taxonomy-tags-section--desktop">
+                {% if node.field_applicable_to.value %}
+                    <span class="wdn-text-hidden">{{ applicable_to['#title'] }}</span>
+                    <ul class="funding-ct--taxonomy-tags">
+                        {% for i, value in applicable_to['#items'] %}
+                            {% set tag_value = applicable_to[i]['#title']|split(':')%}
+                            {% set tag_value_last_index = (tag_value|length - 1) %}
+                            <li><a class="wdn-sans-serif" href="{{ applicable_to[i]['#url'] }}">{{ tag_value[tag_value_last_index] }}</a></li>
+                        {% endfor %}
+                    </ul>
+                {% endif %}
+            </section>
+            <section class="funding--full-description">
+                <a id="full-description" class="anchor"></a>
+                {{ full_description }}
+            </section>
+            <div id="tabs">
+                <ul>
+                    {% if node.field_funding_eligibility.value %}
+                        <li><a href="#eligibility-tab">Eligibility</a></li>
+                    {% endif %}
+                    <li><a href="#how-to-apply-tab">How to Apply</a></li>
+                </ul>
+                {% if node.field_funding_eligibility.value %}
+                    <div id="eligibility-tab">
+                        <h2>Eligibility</h2>
+                        {{ eligibility }}
+                    </div>
+                {% endif %}
+                <div id="how-to-apply-tab">
+                    <h2>How to Apply</h2>
+                    {{ how_to_apply }}
+                </div>
+            </div>
+        </div>
+    </div>
+</article>
+
+
+{#script for JQuery UI tabs#}
+<script>
+    //<![CDATA[
```

Comment:
The "proper" way to add JS is to put it in its own .js file and include it the same way as the .css file (with a definition in the libraries.yml file)